src/main/java/uk/gov/pay/directdebit/mandate/services/MandateUpdateService.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.pay.directdebit.mandate.services;
```

Comment:
I like that we're separating out the functionality into smaller mandate services. Not especially related to this PR but there is `MandateService` which now appears to have creation logic, so that could be named something more expressive like `MandateCreateService`, then we'd have `MandateQueryService`, `MandateCreateService` and `MandateUpdateService`